qddssrc/popemp.sql

```diff
@@ -0,0 +1,63 @@
+create or replace procedure popemp()
+P1: BEGIN
+    declare v_url CLOB(10M);
+    declare v_response CLOB(10M);
+    declare v_dept_name varchar(36);
+    declare v_mgr_no char(6);
+    declare v_admr_dept char(3);
+    declare v_location char(16);
+    declare v_emp_no char(6);
+    declare v_first_name varchar(12);
+    declare v_mid_init char(1);
+    declare v_last_name varchar(15);
+    declare v_work_dept char(3);
+    declare v_phone_no char(4);
+    declare v_hire_date date;
+    declare v_job char(8);
+    declare v_ed_level SMALLint;
+    declare v_sex char(1);
+    declare v_birth_date date;
+    declare v_salary decimal(9,2);
+    declare v_bonus decimal(9,2);
+    declare v_comm decimal(9,2);
+    declare i int default 1;
+
+
+
+    
+    -- Populate EMPLOYEE table
+    while i <= 200 DO
+        set v_url = 'https://randomuser.me/api/?nat=in';
```

Comment:
I see that you mentioned that you hard coded `in` (for India), but perhaps the `nat` property could be an optional parameter on this procedure?

Reply:
Good idea. I have added a parameter for specifying nationality. 

Although, one thing that I noticed is that, if we don't specify nationality then sometimes it would generate names that contains non-Latin alphabets such as Russian or Arabic. That would cause character conversion error. So, it is always best to include the nationality :)